Evergreen/Manifests/Praat.json

```diff
@@ -0,0 +1,22 @@
+{
+	"Name": "Praat",
```

Comment:
Is the vendor "Phonetic Sciences"?

Reply:
There's no entry in winget for this app, but the chocolatey entry lists the authors as Paul Boersma and David Weenink. This is one of those apps that is widely used in academic environments and written by fellow academics. The authors work in the Phonetic Sciences department of the University of Amsterdam - unsure as to what vendor to put here. I've left it as is just now, but can change if required

- https://community.chocolatey.org/packages/praat